app/templates/administrative-units/administrative-unit/legal-structures/new.hbs

```diff
@@ -0,0 +1,145 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <PageHeader>
+      <:title>Voeg juridische structuur toe</:title>
```

Comment:
I think this should be "Voeg **verbonden** juridische structuur toe"

Reply:
You are right. I'll fix it in the "edit" page too in the new edit PR that solve the save bug.